code-of-conduct.md

```diff
@@ -2,47 +2,48 @@
 
```

Comment:
live feedback: should have "you" replaced with "member" or define it or somewhere since this is legal